lib/portlayer/storage/vsphere/datastore.go

```diff
@@ -301,3 +303,36 @@ func (d *Datastore) mkRootDir(ctx context.Context, rootdir string) error {
 func (d *Datastore) rootDir() string {
 	return strings.SplitN(d.RootURL, " ", 2)[1]
 }
+
+// Parse the datastore format ([datastore1] /path/to/thing) to groups.
+var datastoreFormat = regexp.MustCompile(`\[([\w\d]*)\]\ (.*)?`)
```

Comment:
You're regex is stronger than mine.  Lets chat offline and you can help me craft something smarter.